catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/ServiceProperties.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard;
+
+import java.util.Map;
+import java.util.Set;
+import javax.activation.MimeType;
+
+/**
+ * This represents the basic properties associated with transformers.
+ *
+ * <p><b> This code is experimental. While this interface is functional and tested, it may change or
```

Comment:
❗️ Move this interface and the others to an `internal` package and remove this JavaDoc.  Here is an example.

https://github.com/codice/ddf/blob/10ef9296f2c6febf83e0e5223ecf87eea8ea65c6/platform/io/platform-io-internal-api/src/main/java/org/codice/ddf/platform/io/internal/PersistenceStrategy.java#L14

Reply:
done